src/cli/argument-parser.js

```diff
@@ -144,6 +153,7 @@ export default class CLIArgumentParser {
             .option('-T, --test-grep <pattern>', 'run only tests matching the specified pattern')
             .option('-f, --fixture <name>', 'run only fixtures with the specified name')
             .option('-F, --fixture-grep <pattern>', 'run only fixtures matching the specified pattern')
+            .option('--element-timeout <miliseconds>', 'set the amount of time allowed for a page element to become available before the test fails')
```

Comment:
We can use `<ms>` instead of `<milliseconds>` to save horizontal space and avoid line wrap in CLI